src/include/parser/expression/parsed_function_expression.h

```diff
@@ -8,28 +8,29 @@ namespace parser {
 class ParsedFunctionExpression : public ParsedExpression {
 public:
     ParsedFunctionExpression(std::string functionName, std::string rawName, bool isDistinct = false)
-        : ParsedExpression{common::FUNCTION, std::move(rawName)}, isDistinct{isDistinct},
```

Comment:
Can u keep this expression type FUNCTION. I think for parsed expression there is no way to differentiate whether it's scalar or aggregate